BlueDB/src/main/java/org/bluedb/api/BlueCollectionBuilder.java

```diff
@@ -0,0 +1,11 @@
+package org.bluedb.api;
+
+import java.io.Serializable;
+
+import org.bluedb.api.exceptions.BlueDbException;
+
+public interface BlueCollectionBuilder<T extends Serializable> {
+	public BlueCollectionBuilder<T> withName(String name);
+	public BlueCollectionBuilder<T> registerClasses(Class<? extends Serializable>[] additionalRegisteredClasses);

```

Comment:
I like something like `classesToRegister` better than `additionalRegisteredClasses` for the parameter name. Unless we really want them to know that they don't need to register things like standard data structures.

Reply:
that change is made in a later request.